ddtrace/debugging/safety.py

```diff
@@ -71,9 +71,14 @@ def _has_safe_dict(_type):
 
 def _safe_dict(o):
     # type: (Any) -> Dict[str, Any]
-    if _has_safe_dict(type(o)):
```

Comment:
does _has_safe_dict is still used? maybe it can removed?

Reply:
Good point. I thought of leaving just in case, but I guess we can always retrieve it from the history.